model/src/main/java/org/cbioportal/model/GenesetData.java

```diff
@@ -0,0 +1,44 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+
+public class GenesetData implements Serializable {
```

Comment:
Same thing goes for this as well, we can change the current `GeneticData.java` to `GeneData.java` and create an abstract parent for these two called `GeneticData.java`, it can have `geneticProfileId`, `sampleId` and `value`, and we can remove those fields from these two classes.